server/jetstream_api.go

```diff
@@ -262,7 +262,22 @@ const JSMaxNameLen = 256
 // TODO(dlc) - Move to more generic location.
 type ApiError struct {
 	Code        int    `json:"code"`
+	Kind        string `json:"kind,omitempty"`
```

Comment:
We should think if these should be strings or numbers. For direct comparison we would have to look to C and would need numbers, not strings.